src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.sections {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  counter-reset: day-number;
+  flex-wrap: wrap;
+  display: flex;
+  width: $my-var;
+  gap: $width-gap;
+
+  .calendar__day {
+    background-color: #eee;
+    border: 1px solid #000;
+    width: 100px;
```

Comment:
The width of the day box is hardcoded. It would be better to move it to a variable, so that it can be easily changed in one place. Consider creating a variable `$calendar-day-size`.